internal/venice-common/src/main/java/com/linkedin/venice/controllerapi/LeaderControllerResponse.java

```diff
@@ -20,4 +21,12 @@ public String getUrl() {
   public void setUrl(String url) {
     this.url = url;
   }
+
+  public String getSecureUrl() {
```

Comment:
is `secureUrl` a url backed by tls? I wonder its difference between regular url

Reply:
Yes. `secureUrl` is the https url